global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.100"
```

Comment:
Why do we need to pin this?

Reply:
My environment must have been in a bad state. I tried removing the global.json and everything now runs fine on my machine. Previously, when I tried to `dotnet run` this project, it was trying to use a prerelease version of the ASP.NET Core runtime which I don't have.

```
> dotnet run
It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '3.1.0-preview1.19508.20' was not found.
``` 

I don't know why it was choosing that runtime version. I didn't see any reference to it in the project, and I didn't have any other global.jsons further up the directory tree. Explicitly setting the SDK in the global.json did seem to fix the issue for me. @JunTaoLuo was there and can back me up. He was there.

Reply:
Now that I think about it, I did only install version 3.1.100 of the SDK after making the global.json change. That might have been what really fixed things for me.